pkg/schemas/service.go

```diff
@@ -223,6 +227,59 @@ func (s *Service) DeleteSchema(ctx context.Context, ID string) error {
 
 }
 
+func (s *Service) GetDefaultSchema(ctx context.Context) (*DefaultSchema, error) {
+	ctx, span := s.tracer.Start(ctx, "schemas.Service.GetDefaultSchema")
+	defer span.End()
+
+	cm, err := s.k8s.ConfigMaps(s.cmNamespace).Get(ctx, s.cmName, metaV1.GetOptions{})
+
+	if err != nil {
+		s.logger.Error(err.Error())
+		return nil, err
+	}
+
+	ID, ok := cm.Data[DEFAULT_SCHEMA]
+
+	if !ok {
+		return nil, fmt.Errorf("default schema %s missing", DEFAULT_SCHEMA)
+	}
+
+	defaultSchema := new(DefaultSchema)
+	defaultSchema.ID = ID
+
+	return defaultSchema, nil
+}
+
+func (s *Service) UpdateDefaultSchema(ctx context.Context, schema *DefaultSchema) (*DefaultSchema, error) {
```

Comment:
I would say to rename this var, so that it is clear what it is.
```suggestion
func (s *Service) UpdateDefaultSchema(ctx context.Context, schemaID *DefaultSchema) (*DefaultSchema, error) {
```